src/Mod/TechDraw/App/DrawPagePyImp.cpp

```diff
@@ -45,6 +45,31 @@ PyObject* DrawPagePy::addView(PyObject* args)
     return PyInt_FromLong((long) rc);
 }
 
+PyObject* DrawPagePy::removeView(PyObject* args)
+{
+    //this implements iRC = pyPage.removeView(pyView)  -or-
+    //doCommand(Doc,"App.activeDocument().%s.removeView(App.activeDocument().%s)",PageName.c_str(),FeatName.c_str());
+    PyObject *pcDocObj;
+
+    if (!PyArg_ParseTuple(args, "O!", &(App::DocumentObjectPy::Type), &pcDocObj)) {
+        Base::Console().Error("Error: DrawPagePy::removeView - Bad Arg - not DocumentObject\n");
```

Comment:
I consider it a bad idea to write errors to the console because you end up writing the error message twice. If you want a different message than the generic Python message then explicitly write:
PyErr_SetString(PyExc_TypeError, "DrawPagePy::removeView - Bad Arg - not DocumentObject");